src/main/java/seedu/address/storage/XmlAddressBookStorage.java

```diff
@@ -74,4 +85,63 @@ public void saveAddressBook(ReadOnlyAddressBook addressBook, String filePath) th
         XmlFileStorage.saveDataToFile(file, new XmlSerializableAddressBook(addressBook));
     }
 
+    @Override
+    public void backupAddressBook() throws IOException {
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.

Reply:
Noted. Necessary changes have been made by the group.